android/libraries/rib-base-test/src/main/java/com/badoo/common/ribs/RibTestHelper.kt

```diff
@@ -0,0 +1,101 @@
+package com.badoo.common.ribs
+
+import android.os.Parcelable
+import android.view.ViewGroup
+import com.badoo.ribs.core.Interactor
+import com.badoo.ribs.core.Node
+import com.badoo.ribs.core.Rib
+import com.badoo.ribs.core.Router
+import com.badoo.ribs.core.routing.action.RoutingAction
+import com.badoo.ribs.core.view.RibView
+import com.jakewharton.rxrelay2.Relay
+import io.reactivex.ObservableSource
+import io.reactivex.Observer
+import kotlinx.android.parcel.Parcelize
+import org.mockito.ArgumentMatchers.any
+import org.mockito.Mockito.mock
+import org.mockito.Mockito.`when` as whenever
+
+class RibTestHelper<View : RibView>(
+    val interactor: Interactor<*, *, *, View>,
+    val viewFactory: ((ViewGroup) -> View?)? = null,
+    router: Router<*, *, *, *, View>? = null
+) {
+
+    val router: Router<*, *, *, *, View> = router ?: TestRouter.createTestRouter()
+
+    var nodeCreator: () -> Node<View> = {
+        Node(
+            savedInstanceState = null,
+            identifier = TestIdentifier,
+            viewFactory = viewFactory,
+            router = this.router,
+            interactor = interactor
+        )
+    }
+
+    fun resumeAndExecute(block: (Node<View>) -> Unit) {
+        startAndExecute {
+            it.onResume()
+            block(it)
+            it.onPause()
+        }
+    }
+
+    fun startAndExecute(block: (Node<View>) -> Unit) {
```

Comment:
These methods names are not reflecting things happening in the body.
For example, it is not quite clear that the node will be stopped.